source/documentation/index.md

```diff
@@ -0,0 +1,18 @@
+# Overview
+
+[Find MoJ Data](https://data-platform-find-moj-data-prod.apps.live.cloud-platform.service.justice.gov.uk/) is the data catalogue service for the Ministry of Justice. It enables data discoverability across the suite of data assets used by and owned by the Ministry of Justice by providing a searchable interface of their latest metadata. No row-level data is viewable.
```

Comment:
"No row-level data" seems a bit confusing to me...

Could we try something like "This service only stores *metadata* - information about data assets - it doesn't contain any data from the catalogued data assets. (but worded better than this 😅)